docs/fiddles/features/drag-and-drop/main.js

```diff
@@ -1,20 +1,23 @@
 const { app, BrowserWindow, ipcMain, nativeImage, NativeImage } = require('electron')
-const fs = require('fs');
-const http = require('http');
+const path = require('path')
+const fs = require('fs')
+const http = require('http')
 
 function createWindow () {
   const win = new BrowserWindow({
     width: 800,
     height: 600,
     webPreferences: {
-      nodeIntegration: true
+      preload: path.join(__dirname, 'preload.js')
     }
   })
 
   win.loadFile('index.html')
 }
-const iconName = 'iconForDragAndDrop.png';
-const icon = fs.createWriteStream(`${process.cwd()}/${iconName}`);
+
+const iconName = path.join(process.cwd(), 'iconForDragAndDrop.png');
```

Comment:
Should this be `__dirname` instead?

Reply:
These are probably functionally the same in fiddle, but it seems like not in an app. [`__dirname`](https://nodejs.org/docs/latest/api/modules.html#modules_dirname) is the directory for the current js file, and [`process.cwd()`](https://nodejs.org/api/process.html#process_process_cwd) is the working directory for the node.js process. I'm guessing the working directory (`process.cwd()`) is more correct in an application context?

Reply:
I was thinking the opposite because we're assuming the file structure is that `main.js` lives in the same folder as `preload.js`, so `__dirname` guarantees to hit that regardless of where you're calling the command (i.e. what if `main.js` lives in some `src/` subfolder?)

Reply:
Well, both will probably fail in a packaged app on mac, but that's likely overthinking things a bit. To me it makes sense to put it relative to the executable, not to the javascript contents - but I don't have a strong opinion here.

Reply:
Went ahead and changed to `__dirname`.